src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/include/rc_rpc_plugin/mock/mock_resource_allocation_manager.h

```diff
@@ -2,39 +2,37 @@
 #define SRC_COMPONENTS_REMOTE_CONTROL_TEST_INCLUDE_MOCK_RESOURCE_ALLOCATION_MANAGER_H_
 
 #include "gmock/gmock.h"
-#include "remote_control/resource_allocation_manager.h"
+#include "application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/resource_allocation_manager.h"
```

Comment:
@LitvinenkoIra  This path must move into the CMakeLists  ` application_manager/rpc_plugins/rc_rpc_plugin/include/ `